wordpress-mobile/WordPress-iOS
# Migrate WordPressComOAuthClient to Swift

Refs #5245 

To test: 
  - Run the unit tests and see if all of them are passing.
  - Try to login in the app using a WordPress.com sites. 
  - Try with a site with 2FA another without 2FA

Needs review: @koke
